examples/lock-app/silabs/src/AppTask.cpp

```diff
@@ -234,7 +234,10 @@ CHIP_ERROR AppTask::Init()
         GetLCD().ShowQRCode(true);
     }
 #endif // QR_CODE_ENABLED
-#endif
+#ifdef SL_ENABLE_ICD_LCD
```

Comment:
Does this define already exist or is it supposed to be added in a .gn file in this PR?

Reply:
It will be added to the .gn file in this PR, enabled on the sleepy applications as requested.